src/components/AddFavorites/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { ReactComponent as StarIcon } from "../../assets/star.svg";
+import { IAddFavoritesProps } from "../../types/interfaces";
+export default function AddFavorites({
+  backgroundColor,
+  textColor,
+  favorited,
+  toggleFavorite
+}: IAddFavoritesProps) {
+  return (
+    <button className="favorites--button" onClick={toggleFavorite}>
+      <div
+        className="favorites--container"
+        style={{ backgroundColor: `var(${backgroundColor})` }}
+      >
+        <StarIcon
+          style={
+            favorited
+              ? {
+                  fill: `var(${textColor})`,
+                  stroke: `var(${textColor})`
+                }
+              : {
+                  stroke: `var(${textColor})`
+                }
+          }
+        />
+        <div className="favorites--text" style={{ color: `var(${textColor})` }}>
```

Comment:
Add additional classes rather than using style.

Reply:
So should I only target colors provided from the design file? Instead of making it work with every color

Reply:
You can add another property named `fontClassName` and then you can add it to your sub-components classnames.
`<div className={`favorites--text ${fontClassName ? fontClassName : ""}`}>`

Reply:
So developers can send a new classname depending on their needs.

Reply:
I got it thanks!